src/vmspawn/vmspawn.c

```diff
@@ -35,6 +35,7 @@
 #include "pretty-print.h"
 #include "process-util.h"
 #include "rm-rf.h"
+#include "sd-daemon.h"
 #include "sd-event.h"
 #include "sd-id128.h"
```

Comment:
actually, can you move these three headers up? our public headers (i.e. `sd-*.h`; public in the sense that other software can use them, i.e. apis in libsystemd.so) we generally put first, i.e. between the `<>` and our regular `""` headers, if you follow what I am trying to see. See where sd-*.h is included in most other sources in our tree.